scalene/scalene_profiler.py

```diff
@@ -731,7 +731,6 @@ def cpu_signal_handler(
                         to_wait = min(remaining_time, next_interval)
                     else:
                         to_wait = next_interval
-                        Scalene.client_timer.reset()
```

Comment:
This works but I still get the warning about the signal, so we need to filter that.

```
WARNING: Scalene uses Alarm clock: 14 to profile.
If your code raises Alarm clock: 14 from non-Python code, use SIGUSR1.
Code that raises signals from within Python code will be rerouted.
```

Reply:
Right-- we print the warning because, while we can transparently redirect everything in Python-land, we want to warn the user in case they're doing something in C-land